app/parsers/bulkrax/oai_adventist_qdc_parser.rb

```diff
@@ -92,7 +92,12 @@ def build_records
         end
         flatten_arrays(%w[@collections @file_sets @works])
       else # if no model is specified, assume all records are works
-        @works = records
+        # @see https://github.com/scientist-softserv/adventist-dl/issues/208
+        #
+        # In this case `records` is a "OAI::ListRecordsResponse" object which is an Enumerable but
+        # does not respond to flatten.  By using Array() we convert the records object into an actual
+        # array.
+        @works = Array(records)
```

Comment:
Out of curiosity, do you know if there's a difference between `Array(records)` and `Array.new(records)`? I feel like I've seen the latter a few times, but don't remember ever seeing the former

Reply:
`Array(records)` coerces the object into an array.  `Array.wrap(records)` is a Rails idiom that is just a bit smarter (but sometimes wrong in Samvera when coping with ActiveTriples).  And `Array.new(10)`  instantiates and Array with 10 elements (of `nil`); `Array.new(records)` will raise an error.